test/testinput/bufr_ncep_read_2_dim_blocks.yaml

```diff
@@ -47,50 +47,50 @@ observations:
             - "*/QCPRMS"
 
       variables:
-        - name: "originatingGeneratingCenter@MetaData"
+        - name: "MetaData/originatingGeneratingCenter"
           source: variables/originatingGeneratingCenter
           longName: "Originating Generating Center"
           units: "id"
 
-        - name: "pressure@GeneratingApplication"
```

Comment:
What is the "GeneratingApplication" group? It's not in the ObsSpace.yaml or the Conventions document. Are we sure it shouldn't just be the ObsValue group? This goes for variables with the "PercentConfidence" group below, too.

Reply:
@nicholasesposito, this is what we discussed before for this yaml file. I am not sure if Greg can accept them. @emilyhcliu perhaps help me to make an issue to discuss. Right now I just leave them there.

Reply:
@nicholasesposito, if possible, I am able to bring such a yaml file to JEDI IODA Hour and see what other people will recommend.  If you have any thoughts, we can discuss during that meeting

Reply:
This YAML file is a unit test that tests the reading of 2 dimensional blocks... The underlying data (and the naming convetnions) are irrelevant to the purpose of this YAML file (no need to say ncep in the file name). Emily is correct.

Reply:
@emilyhcliu, I modified this PR to remove this yaml file you suggested. Do I need to modify  CMakeLists.txt  in test directory, and copy bufr_read_2_dim_blocks.nc to and bufr_read_2_dim_blocks.yaml from feature/query_cxx to corresponding directories, git add, git comiit, and git push and make my PR clean, right?

Reply:
> @YoulongXia-NOAA This YAML is related to satellite winds. This one is being taken care of in the ioda-converter code sprint. I will take care of this. You can remove this YAML from the PR.

@emilyhcliu , Greg only modify dateTime and units. He did not take care of varibleName**@**somting to something**/**variableName. You may need to modify them. This is what I found. Please double check.